guides/V1_space_template_main.md

```diff
@@ -18,4 +18,4 @@ as well as a set of required apps that are automatically added to the space on c
 | status |SpaceStatus |Status is a top-level list style attribute on the template. Status consists of a list of values that can be used to describe the state of the space; for example, open, closed, archived.|
 | requiredApps |SpaceRequiredApplicationCollection |Required apps define a set of applications (by ID) that are added to each new space on creation.|
 | teamId |String |The id of the team in which the space will be created, derived either from the offering with which the template is associated (in the case of an offering template) or from the user who created the template (in the case of a user-created custom template)|
-| offeringCollaborationType |OfferingCollaborationType |The type of offering with which the space will be associated, derived either from the offering with which the template is associated (in the case of an offering template) or from the user who created the template (in the case of a user-created custom template)|
+| offeringCollaborationType |OfferingCollaborationType |An offering defines the capabilities granted to users with a subscription to a given product. There are two broad types of offering: PERSONAL and TEAM. A user with a subscription to a TEAM offering belongs to a team. The type of offering with which the space will be associated, derived either from the offering with which the template is associated (in the case of an offering template) or from the user who created the template (in the case of a user-created custom template)|
```

Comment:
I would swap this around and add the new explanation of an offering after the existing text.

| property      | type          | description  |
| ------------- |:------------- |:-----|
| offeringCollaborationType |OfferingCollaborationType | The type of offering with which the space will be associated, derived either from the offering with which the template is associated (in the case of an offering template) or from the user who created the template (in the case of a user-created custom template). An offering defines the capabilities granted to users with a subscription to a given product. The two broad types of offering are `PERSONAL` and `TEAM`. A user with a subscription to a `TEAM` offering belongs to a team. |